lib/dispatcher.lib.php

```diff
@@ -369,6 +369,14 @@ public function uploadreplay($dispatcher, &$reqData, &$out) {
 		die($GLOBALS['Replays']->upload($reqData));
 	}
 
+	public function fetchreplay($dispatcher, &$reqData, &$out) {
```

Comment:
...lol

I made a CORS change so you wouldn't have to roundtrip through `dispatcher` anymore.